src/main/groovy/com/github/spotbugs/snom/SpotBugsExtension.groovy

```diff
@@ -40,8 +40,8 @@ import org.gradle.api.provider.Property;
  * &nbsp;&nbsp;&nbsp;&nbsp;visitors = [ 'FindSqlInjection', 'SwitchFallthrough' ]<br>
  * &nbsp;&nbsp;&nbsp;&nbsp;omitVisitors = [ 'FindNonShortCircuit' ]<br>
  * &nbsp;&nbsp;&nbsp;&nbsp;reportsDir = file("$buildDir/reports/spotbugs")<br>
- * &nbsp;&nbsp;&nbsp;&nbsp;includeFilter = file('spotbugs-include.xml')<br>
- * &nbsp;&nbsp;&nbsp;&nbsp;excludeFilter = file('spotbugs-exclude.xml')<br>
+ * &nbsp;&nbsp;&nbsp;&nbsp;includeFilter = file('spotbugs-include.xml') or new URL('http://sonar?findbugs-include.xml')<br>
```

Comment:
could you use comments to explain the usage of URL, or this snippet becomes invalid groovy script so user cannot copy-and-paste.

Reply:
So you would want the '...or new URL' part removed from here and added below instead?

Reply:
So you would want the '...or new URL' part removed from here and added below instead?